TWLight/applications/templates/applications/apply.html

```diff
@@ -20,4 +20,11 @@
       </div>
     </div>
   </div>
+  {% if partner.authorization_method == EMAIL %}
+    If your application is successful we may share your email address with {{ partner }}. This is required to set up your 
+    access to their resources. By submitting this form you agree to allow your email address to be shared with {{ partner }} 
+    if your application is successful, for the purposes of account setup.
+  {% endif %}
+
+
```

Comment:
This text should have translation tags `{% blocktranslate trimmed %}`. It should also be a bootstrap alert.

```suggestion
{% if partner.authorization_method == EMAIL %}
    {% comment %}Translators: This is a message shown when a user tries to apply for a collection where their email will be shared with the partner.{% endcomment %}
    <div class="alert alert-secondary" role="alert">
        {% blocktranslate trimmed %}
            If your application is successful we may share your email address with {{ partner }}. This is required to set up your access to their resources. By submitting this form you agree to allow your email address to be shared with {{ partner }} if your application is successful, for the purposes of account setup.
        {% endblocktranslate %}
    </div>
{% endif %}
```